content/docs/reference/install.md

```diff
@@ -224,9 +224,9 @@ warning: A new version of Pulumi is available. To upgrade from version '0.17.26'
 or visit https://pulumi.com/docs/reference/install/ for manual instructions and release notes.
 ```
 
-<div class="note note-warning" role="alert">
+<div class="note note-info" role="alert">
     <p>
-        <span>Skip version check.</span> If you're in an environment with no internet access, you may skip the Pulumi version update check by setting the environment variable <code>PULUMI_SKIP_UPDATE_CHECK</code> to <code>1</code> or <code>true</code>.
+        <span>Skip version check:</span> If you're in an environment with no internet access, you may skip the Pulumi version update check by setting the environment variable <code>PULUMI_SKIP_UPDATE_CHECK</code> to <code>1</code> or <code>true</code>.
```

Comment:
Semantically, these should probably be `<strong>`s rather than `<span>`s. The `<code>` backgrounds also get a bit lost in these, with their default styling. (I'm happy to jump in and do this.)

I'd also sort of strongly prefer we try to avoid writing HTML into these content files, and opt for shortcodes or partials instead.

Reply:
Would it make sense to add this as a shortcode when it's only applicable to this page? At least for now anyway. I agree with avoiding HTML on the content files however, so I'd be ok with moving this to a shortcode.